tests/contract_testcases/solana/selector_override.sol

```diff
@@ -1,8 +1,26 @@
 contract selector {
-	constructor() selector=hex"abcd" {}
-	modifier m() selector=hex"" {_;}
-	receive() payable external selector=hex"1" {}
-	fallback() external selector=hex"abc" {}
-	function i() internal selector = hex"ab_dd" {}
-	function p() private selector = hex"ab_dd" {}
+	@selector([0xab, 0xcd])
+	constructor() {}
+	@selector([x])
+	modifier m() {_;}
+	@selector([1])
+	receive() payable external {}
+	@selector([0xabc])
+	fallback() external {}
+	@selector([0xab, 0xdd])
+	function i() internal {}
+	@selector([0xab, 0xdd])
+	function p() private {}
+}
+
+contract TooLarge {
+    @selector([1, 2, 3, 256])
+    function get_foo() pure public returns (int) {
+        return 102;
+    }
+
+    @selector([0x05, 0x06, 0x07, 0xab8])
```

Comment:
`0xab8` appears as `2744` in the error message. Is that OK?

Reply:
Yes, that's fine.